test/multi-stream.test.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+var fs = require('fs')
+var test = require('tap').test
+var mspino = require('../multi-stream')
+
+test('sends to multiple streams', function (t) {
+  var streams = [
+    {stream: fs.createWriteStream('/tmp/info.out')},
+    {level: 'debug', stream: fs.createWriteStream('/tmp/debug.out')},
+    {level: 'fatal', stream: fs.createWriteStream('/tmp/fatal.out')}
```

Comment:
Even better, you might use flush-write-stream (or a Writable) and check the content with that.